page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/utils/Wait.java

```diff
@@ -0,0 +1,637 @@
+package ru.sbtqa.tag.pagefactory.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.aeonbits.owner.ConfigFactory;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import ru.sbtqa.tag.pagefactory.environment.Environment;
+import ru.sbtqa.tag.pagefactory.properties.Configuration;
+
+import static java.lang.String.format;
+
+public class Wait {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=111&issues=AWdUlBboy5Us1olWVmP3&open=AWdUlBboy5Us1olWVmP3)

[ISSUE_KEY]: AWdUlBboy5Us1olWVmP3